File: pgm/ABC_Visual_Basic/ABC_137_B.vb
' Imports System.Collections.Generic
' Imports System.Globalization
' Imports System.IO
' Imports System.Linq
' Imports System.Text
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim K, X As Integer
        Dim arr() As String
        arr = Split(Console.ReadLine())
        K = arr(0) : X = arr(1)
        Dim ans = new List(Of Integer)()
        For i As Integer = X - K + 1 To X + K - 1
            ans.Add(i)
        Next
        Console.WriteLine(String.join(" ", ans))
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_137_B.cs
using System;
using System.Collections.Generic;
// using System.Linq;
// using System.Text;

class Program
{
    static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        int K, X;
        K = int.Parse(arr[0]);
        X = int.Parse(arr[1]);
        List<int> ans = new List<int>();
        for (int i = X - K + 1; i < X + K; i++) ans.Add(i);
        Console.WriteLine(String.Join(" ", ans));
        return;
    }
}